build.yaml

```diff
@@ -2732,6 +2732,38 @@ steps:
       - workdir_image
       - hail_ubuntu_image
       - gpu_image
+  - kind: runImage
+    name: test_auth
+    numSplits: 1
+    image:
+      valueFrom: auth_image.image
+    script: |
+      set -ex
+
+      hail-pip-install -r /io/dev-requirements.txt
+
+      python3 -m pytest \
+              -Werror:::hail -Werror:::hailtop -Werror::ResourceWarning \
+              --log-cli-level=INFO \
+              -s \
+              -r A \
+              -vv \
+              --instafail \
+              --durations=50 \
+              --log-date-format="%Y-%m-%dT%H:%M:%S" \
+              --log-format="%(asctime)s %(levelname)s %(name)s %(filename)s:%(lineno)d:%(funcName)s %(message)s" \
+              -k "not test_scale and not test_invariants" \
```

Comment:
do you need this line as these test names don't appear in your new test module?

Reply:
Good catch. This is copy/pasted from the batch test suite.